src/LibraryViewExtension/LibraryViewController.cs

```diff
@@ -99,8 +100,28 @@ internal LibraryViewController(Window dynamoView, ICommandExecutive commandExecu
 
             this.commandExecutive = commandExecutive;
             InitializeResourceStreams(dynamoViewModel.Model, customization);
+            dynamoWindow.StateChanged += DynamoWindowStateChanged;
+            dynamoWindow.SizeChanged += DynamoWindow_SizeChanged;
         }
 
+        //if the window is resized toggle visibility of browser to force redraw
+        private void DynamoWindow_SizeChanged(object sender, SizeChangedEventArgs e)
+        {
+            this.browser.Visibility = Visibility.Hidden;
+            this.browser.Visibility = Visibility.Visible;
```

Comment:
just curious why `InvalidateVisual` is not useful here?

Reply:
I'll try this instead, I thought I tried it but perhaps tried something else.

Also `SetOffScreenRenderingBestPerformanceArgs` causes an exception when used :(.

Reply:
hmm, I'll give `SetOffScreenRenderingBestPerformanceArgs` another shot as well and disable the manual gpu disable.

Reply:
But I am not sure whether this will fix the black screen thing..after reading the section, looks like this will reduce the CPU usage..and webgl will be disabled.

Reply:
yep, it's already disabled as gpu disable is passed everywhere. It's hard to tell whats actually fixing what as I'm making multiple changes, but testing this more thoroughly may require more time than we're trying to bound this by.

Have you tested locally yet? I will push another change in a few minutes.